.github/workflows/run-tests.yml

```diff
@@ -0,0 +1,72 @@
+name: Tests
+
+on: [push, pull_request]
+
+jobs:
+  test:
+    runs-on: ubuntu-16.04
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest]
+        php: [5.4, 8.0, 7.4, 7.3, 7.2, 7.1, 7.0, 5.6, 5.5]
+        imagemagic: [
+          7.0.10-27,
+          7.0.8-4,
+          7.0.1-0,
+          git7,
+          6.9.2-0,
+          6.8.7-0,
+          6.7.8-0,
+          git6
+        ]
+        exclude:
+          - php: 5.4
+            imagemagic: 6.8.7-0
+        allow_failures:
+          - imagemagic: git7
+          - imagemagic: git6
```

Comment:
shouldn't all `imagemagic` be changed to `imagemagick`, or `imagick` considering it's the project name/

Reply:
😂 where it was so consistent (and I'm a brit), I totally missed that. Good catch.

Reply:
@Lewiscowles1986: your origin does not matter, it's not a word in British nor American English.

Reply:
Good catch, updated!